### PR TITLE
Clean up prelude a bit

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @sdroege @EPashkin

### CODE REVIEW

**Comment by sdroege** [comment]
Only change is to move things? :+1: then

**Comment by GuillaumeGomez** [comment]
Yes. :)

File: src/prelude.rs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// Copyright 2013-2018, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+//! Traits and essential types intended for blanket imports.
```
Comment (EPashkin) [inline src/prelude.rs]: This made travis fail

**Comment by EPashkin** [comment]
@GuillaumeGomez Thanks 👍

**Comment by GuillaumeGomez** [comment]
Then I just merge it. Weird test though...

**Comment by EPashkin** [comment]
I not weird, just checks that we add all comments to https://github.com/gtk-rs/lgpl-docs

**Comment by EPashkin** [comment]
.. or ignored "rt.rs" in build script

**Pull request merged**